pl.nieto.fityk.yaml

```diff
@@ -0,0 +1,114 @@
+id: pl.nieto.fityk
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: fityk
+rename-icon: fityk
+rename-desktop-file: fityk.desktop
+rename-appdata-file: fityk.appdata.xml
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
```

Comment:
this won't work since you force gtk2 with wxwidgets. gtk2 does NOT support wayland.

Reply:
Building wxwidgets with gtk3 doesn't work?

Reply:
The author chose to make it fail the build https://github.com/wojdyr/fityk/commit/85ea545db65d7c6fbb94988b85f1e8cf1c9cbba5 but it seems the problems with Gtk3 are not that major as I thought.